src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.xml

```diff
@@ -11,6 +11,10 @@
             <tag name="cache.pool" clearer="cache.app_clearer" reset="reset" />
         </service>
 
+        <service id="cache.app.simple" class="Symfony\Component\Cache\Simple\Psr6Cache" public="true">
```

Comment:
should it be private?

Reply:
I just kept it consistent with the `cache.app` service which is also public.

So not sure :blush: 

If we don't really promote this service via the documentation I guess we could also make it private.